external/java/src/main/java/ch/epfl/dedis/lib/byzcoin/contracts/DarcInstance.java

```diff
@@ -15,8 +15,8 @@
 import java.util.List;
 
 /**
- * DarcInstance represents an instance of a darc on Omniledger. It is self-
- * sufficient, meaning it has a link to the omniledger instance it runs on.
+ * DarcInstance represents an instance of a darc on ByzCoin. It is self-
+ * sufficient, meaning it has a link to the byzcoin instance it runs on.
```

Comment:
Fixed where I ran across it in the other fix. It was in fact IntelliJ that screwed up this to begin with, because when I renamed the directory, it offered to rename in comments, which I took, and then it lowercased everything. :(